manifests/config.pp

```diff
@@ -70,5 +97,25 @@
       notify  => Class['confluence::service'],
     }
 
+    file { "${confluence::webappdir}/confluence/WEB-INF/web.xml":
+      source  => 'puppet:///modules/confluence/web.xml',
+      mode    => '0600',
+      owner   => $::confluence::user,
+      group   => $::confluence::group,
+      require => Class['confluence::install'],
+      notify  => Class['confluence::service'],
+    }
+
+  }
+  # if JDBC was configured along with the license key and server_id, skip some server setup steps.
+  if $confluence::tomcat_jdbc_settings and $confluence::license and $confluence::server_id {
+    file { "${confluence::homedir}/confluence.cfg.xml":
```

Comment:
They aren't defaults so much as the values the user is supposed to configure via the main class. That is, the confluence class is the public API, and the config class is an implementation detail.

Users are not expected to configure the params or config classes directly (eg. via hiera)